src/components/PriceBar/index.tsx

```diff
@@ -14,7 +14,7 @@ const PriceBar: React.FC = () => {
       bottom="42px"
       right="0px"
       width="calc(100% - 98px)"
-      zIndex="1600"
+      zIndex="1500"
```

Comment:
Same here..

Reply:
Answering the question about the z-index changes in both PriceBar & Editfooter components.
1. The shadow of the PriceBar is not shown in the main branch, and we did not consider the shadow as part of the PriceBar. I also checked the Figma UI design for the court, there was no shadow for the PriceBar as well. The shadow is the box-shadow for the EditFooter, it was covered by the PriceBar in the main branch, using the z-index difference.
2. In the design of the ShoppingCart component, we consider overlapping the shopping cart on the main board interface but the z-indexing of the original design limited us. The original design has an order of the interface stack as EditFooter(z:1500),  PriceBar(z:1600),  The logging overlay(z:1600). The PriceBar and overlay had the same z-index. If we want to add the shopping cart interface, we have to add it between all main court components and the overlay, therefore, we have to reduce the z-index of PriceBar.  In order to keep the EditFooter component behind the PriceBar, we had to reduce the z-index for the EditFooter as well.